to-do-list/src/ListTodo.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import Table from 'react-bootstrap/Table';
+
+import Button from 'react-bootstrap/Button';
+import ListItem from './ListItem';
+
+const ListTodo = ({list, updateCompleted, deleteCompletedItems}) => {
+    const [ids, setIds] = useState([]);
+    const [checked, setChecked] = useState("disabled");
+
+    return <>
+        <Table style={{width:"80%"}} striped bordered hover>
+            <tbody>
+                <tr>
+                    <th></th>
+                    <th>Name</th>
+                    <th>Status</th>
+                    <th>Action</th>
+                </tr>
+                {
+                    list.length > 0 ?
+                    list.map((item)=>{

```

Comment:
if code block is small and  its the only codeblock you are returning , you can avoid return statement
list.map(item => ( <tr key={item.id}>
    <ListItem
      item={item}
      updateCompleted={updateCompleted}
      deleteCompletedItems={deleteCompletedItems}
      setIds={setIds}
      ids={ids}
      setChecked={setChecked}
    />
  </tr>
));